sdk/monitor/Microsoft.OpenTelemetry.Exporter.AzureMonitor/tests/Microsoft.OpenTelemetry.Exporter.AzureMonitor.Integration.Tests/FunctionalTests/AzureMonitorLogExporterLiveTests.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.OpenTelemetry.Exporter.AzureMonitor.Integration.Tests.FunctionalTests
+{
+    using System.Threading.Tasks;
+
+    using global::Azure.Core.TestFramework;
+
+    using global::OpenTelemetry;
+    using global::OpenTelemetry.Logs;
+
+    using Microsoft.Azure.ApplicationInsights.Query;
+    using Microsoft.Extensions.DependencyInjection;
+    using Microsoft.Extensions.Logging;
+
+    using NUnit.Framework;
+
+    /// <summary>
+    /// Collection of tests to evaluate the <see cref="AzureMonitorLogExporter"/>.
+    /// </summary>
+    public class AzureMonitorLogExporterLiveTests : AzureMonitorTestBase
+    {
+        public AzureMonitorLogExporterLiveTests(bool isAsync) : base(isAsync, RecordedTestMode.Record)
+        {
+        }
+
+        [RecordedTest]
+        public async Task VerifyLogExporter()
+        {
+            // SETUP
+            var exporter = this.GetAzureMonitorLogExporter();
+            var processor = new BatchExportProcessor<LogRecord>(exporter);
+
+            var serviceCollection = new ServiceCollection().AddLogging(builder =>
+            {
+                builder.SetMinimumLevel(LogLevel.Trace)
+                    .AddOpenTelemetry(options => options
+                        .AddProcessor(processor));
+            });
+
+            using var serviceProvider = serviceCollection.BuildServiceProvider();
+            var logger = serviceProvider.GetRequiredService<ILogger<AzureMonitorLogExporterLiveTests>>();
+
+            // ACT
+            var testMessage = "Hello World";
+
+            // TODO: For proper test isolation, we should include a CustomProperty on telemetry items.
+            // CustomProperty should be unique per test method.
+            // This would allow us to run tests in parallel.
+            logger.Log(logLevel: LogLevel.Information, message: testMessage);
+
+            processor.ForceFlush();
```

Comment:
Minor, probably put a timeout on `ForceFlush`, and fail the test if `ForceFlush` returned `false`?